server/util/oodlet.js

```diff
@@ -24,7 +24,8 @@ function nextDueDate() {
         nextDate = moment(nextDate).add(2, 'weeks').toDate();
       }
       return nextDate;
-    });
+    })
+    .catch(() => moment().add(2, 'weeks').toDate());
```

Comment:
This part - `.orderBy({index: r.desc('dueDate')})` . Because if there is no oodlet in db rethink throws an err since it cant order them by dueDate.